src/sentry/services/hybrid_cloud/user/impl.py

```diff
@@ -283,6 +287,10 @@ def get_user_avatar(self, *, user_id: int) -> RpcAvatar | None:
 
         return serialize_user_avatar(avatar=possible_avatar)
 
+    def get_bulk_contact_info(self, *, user_ids: list[int]) -> list[RpcUserContact]:
+        users = User.objects.filter(id__in=user_ids).values_list("id", "name", "email")
+        return [RpcUserContact(id=id, name=name, email=email) for (id, name, email) in users]
```

Comment:
I wonder if we have other flows in the application that could benefit from loading less data when needing to show lists of users 🤔

Reply:
Attempted to generalize this in https://github.com/getsentry/sentry/pull/68847.